### DISCUSSION TITLE
What Access Rules are needed for CRL renewal

### DISCUSSION THREAD

**Discussion by eryabies** [Keyfactor/ejbca-ce#185] [Q&A]
Hi!

I'm looking the right Access Rules for the CRL publisher role. 
`/ca_functionality/view_ca/` - Allow
`/ca/<All_my_CAs>/` - Allow
`/ca_functionality/create_crl/` - Allow

Are these rules enough for CRL publisher (list CAs & renew CRLs) or do I need add anything else?

Thanks

**Comment by primetomas** [comment]
It's easy to test. If some access rule is missing when you try to perform this action in the admin UI the log will tell you exactly what is missing.
